solar_consumer/data/fetch_gb_data.py

```diff
@@ -4,10 +4,20 @@
 import json
 import os
 from loguru import logger
-
 from datetime import datetime, timedelta, timezone
 
 from pvlive_api import PVLive
+from solar_consumer.data.nighttime import make_night_time_zeros
+
+# Prefer loading locations from the canonical datamodel; fall back to empty frame
+try:
+    from nowcasting_datamodel.read.gsp import get_all_locations  # type: ignore
+    _GSP_LOCATIONS = get_all_locations().set_index("gsp_id")
```

Comment:
sorry, could we not use `nowcasting_datamodel`, i think copying in a csv of the centeroid of the gsps would be fine